.github/workflows/update-crates-versions.yml

```diff
@@ -0,0 +1,38 @@
+name: Update crates versions
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  schedule:
+    # Every Sunday at 00:00
+    - cron:  '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  updateversions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+        with:
+          fetch-depth: 1
+      - name: Setup rust toolchain
+        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
+        with:
+          toolchain: stable
+          default: true
+      - name: Retrieve newest versions of dependencies
+        run: ./bin/update-crates-versions.sh
+      - name: Create pull request
+        run: |
+          git config --global user.email "githubbot@exercism.io"
+          git config --global user.name "Exercism Bot"
```

Comment:
@saschamann will this work with the GITHUB_TOKEN passed in not being the token of the Exercism bot? And does it even need to be the Exercism Bot user here?

Reply:
It might show up as unverified but generally the git name and email can be whatever. For consistency, you might want to set these to match Exercism Bot elsewhere, or at least point it either at an obvious dummy email address or one that actually works, not one that looks like it may work but likely doesn't.

https://github.com/exercism/org-wide-files/blob/601d9a5924d470499451e441371f9b3bd11caef0/.github/workflows/sync-rest.yml#L23-L24